### ISSUE TITLE
Low level tile rendering

### ISSUE THREAD

**Issue by jordyno** [google/marzipano#227]
I noticed one rendering bug (which might be difficult to fix): [on this demo](http://marzipano-test.360maps.net/?github) :  immediately after opening the page, as only the lower level tiles are loaded, when looking quickly straight up, I can see grey lines around the top cube until higher resolution tiles are loaded. If you don't see it, try to limit your browser's internet speed.
I assume this is caused by blending edges of a low resolution tile on a black background.
It's not that big of an issue, but in some usage scenarios it might cause problems.

**Comment by tjgq** [comment]
I believe the edges are caused by bleeding of the adjacent pixels in the preview cubemap (the single image with all 6 cubefaces in it). They are also visible in the `sample-tour` demo at low resolution (calling `layer.setFixelLevel(0)` helps reproduce).

There's more than one place where this could happen, but after some testing, I'm reasonably sure the culprit is the use of JPEG (which is notoriously bad at preserving edges). As a data point in support of this theory, the bleeding goes away when a PNG image is used for the preview.

Unfortunately, I don't think there's a solution other than not using a single-image preview.

**Comment by jordyno** [comment]
> I believe the edges are caused by bleeding of the adjacent pixels in the preview cubemap (the single image with all 6 cubefaces in it). They are also visible in the `sample-tour` demo at low resolution (calling `layer.setFixelLevel(0)` helps reproduce).
> 
> There's more than one place where this could happen, but after some testing, I'm reasonably sure the culprit is the use of JPEG (which is notoriously bad at preserving edges). As a data point in support of this theory, the bleeding goes away when a PNG image is used for the preview.
> 

I will test this some time later on and report how it works with PNG tiles at multiple layers.

> Unfortunately, I don't think there's a solution other than not using a single-image preview.

I guess that would work only if the single image preview level has `fallbackOnly` set to false in `data.js`?

**Comment by jordyno** [comment]
I still don't understand why is it related specifically to single-image preview, as that one has in this case the `fallbackOnly` set to `true`, which means it should be used only if no higher level tiles are available, if I understand it correctly?

**Comment by tjgq** [comment]
Yes, that is correct. I don't think the issue occurs with non-preview levels, so in normal operation this artifact is only visible until larger levels are loaded.

I would generally recommend against using PNG tiles - they won't compress as well. That was only to verify the assumption that the issue is JPEG-related.
